Source/Widgets/CesiumWidget/CesiumWidget.css

```diff
@@ -18,6 +18,7 @@
   font-size: 10px;
   text-shadow: 0px 0px 2px #000000;
   padding-right: 5px;
+  user-select: none;
```

Comment:
I think its arguable that the credits shouldn't be selectable. It's [possible for users to add additional credits that show onscreen](https://cesium.com/learn/cesiumjs/ref-doc/Credit.html).

Reply:
Fair enough. Yeah I wasn't 100% sure that this was needed for the credits.